### PR TITLE
Target net5.0

### PR DESCRIPTION
**Pull request by WenceyWang**

### CODE REVIEW

File: tests/libvirt.Tests/libvirt.Tests.csproj [hunk 1]
```diff
@@ -11,11 +11,17 @@
...
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.8.0" />
+    <PackageReference Include="Moq" Version="4.15.1" />
+    <PackageReference Include="xunit" Version="2.4.1" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.3">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
+    <PackageReference Include="coverlet.collector" Version="1.3.0">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
```
Comment (falox) [inline tests/libvirt.Tests/libvirt.Tests.csproj:23]: This is for my knowledge: are these attributes needed by net5.0?

**Review by falox — COMMENTED** [review]
PR is ok

**Review by falox — APPROVED** [review]

**Comment by falox** [comment]
@WenceyWang I'm not sure if I should merge into `main` or a separate branch. net5.0 is just released and *is not LTS* (dotnetcore3.1 is)

What's your use case? Why you need net5.0?

**Comment by WenceyWang** [comment]
I think this project is in in early alpha and have many things to do, and does not need to care about ABI compatibility.
`net5.0` comes with a lot of new feature and replaces `netstandard`. it's better for class library.

File: tests/libvirt.Tests/libvirt.Tests.csproj (hunk 1 above)
Comment (WenceyWang) [reply tests/libvirt.Tests/libvirt.Tests.csproj:23]: This is a dev time dependency and is generated by `Update-Package`.

**Pull request merged**